Added more details for how SUSE Edge uses e-c-o (#37)
suse-edge/endpoint-copier-operator: README.md

More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)

## How SUSE Edge uses endpoint-copier-operator

> The first SUSE Edge RKE2/K3s control plane node is deployed with an extra `--tls-san` parameter for an extra IP (and "hostname") that will be used to expose the Kubernetes API. That parameter instructs RKE2/K3s to create the Kubernetes API certificates with that extra IP and hostname. To be able to deploy MetalLB to perform the load balacing, the default 'servicelb' service is disabled.

Reviewer: ```suggestion
The first SUSE Edge RKE2/K3s control plane node is deployed with an extra `--tls-san` parameter for an extra IP (and "hostname") that will be used to expose the Kubernetes API. That parameter instructs RKE2/K3s to create the Kubernetes API certificates with that extra IP and hostname. To be able to deploy MetalLB to perform the load balancing, the default 'servicelb' service is disabled.
```

> Note: those two things are performed automatically via combustion at installation time via edge-image-builder when adding >1 hosts to the kubernetes section of the EIB configuration file.

Reviewer: ```suggestion
Note: those two things are performed automatically via combustion at installation time via edge-image-builder when adding >1 hosts to the Kubernetes section of the EIB configuration file.
```

Then MetalLB is deployed as well as an IPAddressPool and the L2Advertisement (or the BGPAdvertisment) for the K8s VIP.

> On RKE2/K3s the default `kubernetes` service endpoints are the 'Ready' control plane nodes IPs, so an extra `kubernetes-vip` service (type: loadbalancer) is created to behave just like the default kubernetes service does. E-C-O keeps in sync the `kubernetes-vip` endpoint-slices (the `kubernetes-vip` service is created automatically if using edge-image-builder) with the default `kubernetes` service. In the event of a control-plane node going down, it goes down on both `kubernetes` and `kubernetes-vip` services, and if a new control-plane node goes up, it will be reflected as well.

Reviewer: ```suggestion
On RKE2/K3s the default `kubernetes` service endpoints are the 'Ready' control plane nodes IPs, so an extra `kubernetes-vip` service (type: loadbalancer) is created to behave just like the default `kubernetes` service does. E-C-O keeps in sync the `kubernetes-vip` endpointslices (the `kubernetes-vip` service is created automatically if using edge-image-builder) with the default `kubernetes` service. In the event of a control-plane node going down, it goes down on both `kubernetes` and `kubernetes-vip` services, and if a new control-plane node goes up, it will be reflected as well.
```
